### PR TITLE
Replace 'ERR_EXPLAIN' with 'ERR_FAIL_*_MSG' in 'modules/mono'

### PR DESCRIPTION
**Pull request by neikeq**
Replace `ERR_EXPLAIN` with `ERR_FAIL_*_MSG` (and `CRASH_*_MSG` as well) in _'modules/mono'_.

See #31207

### CODE REVIEW

File: modules/mono/csharp_script.cpp [hunk 1]
```diff
@@ -3203,12 +3178,12 @@ int CSharpScript::get_member_line(const StringName &p_member) const {
...
-	if (ferr != OK) {
-		if (ferr == ERR_INVALID_DATA) {
-			ERR_EXPLAIN("Script '" + p_path + "' contains invalid unicode (utf-8), so it was not loaded. Please ensure that scripts are saved in valid utf-8 unicode.");
-		}
-		ERR_FAIL_V(ferr);
-	}
+
+	ERR_FAIL_COND_V_MSG(ferr != OK, ferr,
+			ferr == ERR_INVALID_DATA ?
+					"Script '" + p_path + "' contains invalid unicode (utf-8), so it was not loaded."
```
Comment (akien-mga) [inline modules/mono/csharp_script.cpp]: I'd capitalize "UTF-8".

File: modules/mono/editor/script_class_parser.cpp [hunk 2]
```diff
@@ -657,12 +657,12 @@ Error ScriptClassParser::parse_file(const String &p_filepath) {
...
-	if (ferr != OK) {
-		if (ferr == ERR_INVALID_DATA) {
-			ERR_EXPLAIN("File '" + p_filepath + "' contains invalid unicode (utf-8), so it was not loaded. Please ensure that scripts are saved in valid utf-8 unicode.");
-		}
-		ERR_FAIL_V(ferr);
-	}
+
+	ERR_FAIL_COND_V_MSG(ferr != OK, ferr,
+			ferr == ERR_INVALID_DATA ?
+					"File '" + p_filepath + "' contains invalid unicode (utf-8), so it was not loaded."
```
Comment (akien-mga) [inline modules/mono/editor/script_class_parser.cpp]: Same here.

**Comment by akien-mga** [comment]
Looks good. Could you add dots at the end of error messages? See https://github.com/godotengine/godot/issues/31207#issuecomment-519816715

**Comment by neikeq** [comment]
Done :+1:

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**